src/util/sanitizeSVG.ts

```diff
@@ -0,0 +1,36 @@
+import jsdom from 'jsdom';
+import isSVG from 'is-svg';
+
+const {JSDOM} = jsdom;
+const dom = new JSDOM();
+
+export const sanitizeSVG = (content: Buffer | string): string => {
+    const original = Buffer.isBuffer(content) ? content.toString('utf-8') : content;
+    if (!isSVG(original)) {
+        throw new Error('Not a valid SVG.');
+    }
+
+    const div = dom.window.document.createElement('div');
```

Comment:
The most robust approach I found is to use `jsdom` to actually render the SVG in this safe environment and use DOM APIs to find and remove malicious data.

Alternatively, we could use regexp and string parsing to sanitize the SVG without the additional overhead of `jsdom`. I think the approach with `jsdom` is more straight-forward and less error-prone. But please let me know if you think otherwise or think `jsdom` is going to make it much slower (I haven't noticed significant changes).